frameworks/kafka/docs/upgrade.md

```diff
@@ -106,4 +110,43 @@ Since protocol and log versions have been updated with new protocol and log form
 **Note**: Default settings for protocol and log versions are overwritten with these customized options (steps 1 through 3), even though  their values are same as the defaults of the new package version `2.0.0-0.11.0.0` . Pay attention to these customized options for further package upgrades since they will be preserved unless overwritten explicitly.
 
     
-    
+##  Upgrade in Strict Mode
+
+If you are upgrading to the DC/OS Apache Kafka package `2.0.0-0.11.0.0`  on a DC/OS cluster running in [strict security mode](https://docs.mesosphere.com/1.9/security/#security-modes), you must specify service account details during the package update process. Service account credentials enable your service to authenticate to a DC/OS cluster in strict mode.
+
+
+You can provide service account details in a JSON options file or via the DC/OS GUI.
+
+
```

Comment:
suggest a header and sentence edits:
### New Installation
If you are performing a fresh installation of Kafka `2.0.0-0.11.0.0` on a strict mode cluster...